src/lib/knowledge.ts

```diff
@@ -1,12 +1,11 @@
-import { get } from "svelte/store";
-import { knowledgeStore } from "$lib/store";
-import { generateStorageId } from "./utils";
+import { get } from 'svelte/store';
+import { knowledgeStore } from '$lib/store';
 
 export interface Knowledge {
 	id: string;
 	name: string;
 	content: string;
-  updatedAt: string;
+	updatedAt: string;
```

Comment:
`updatedAt` needs to be a conditional variable, otherwise it breaks the **Knowledge** section for users that created knowledge before this PR is merged.

```suggestion
	updatedAt?: string;
```

Reply:
I agree, but Knowledge's field `updatedAt` was not added in this PR. I checked at the history of commits and that field was always mandatory for Knowledge.